File: src/main/java/com/wafflestudio/draft/security/oauth2/client/exception/SucceedOAuthUserNotFoundException.java
package com.wafflestudio.draft.security.oauth2.client.exception;

import org.springframework.security.core.AuthenticationException;

public class SucceedOAuthUserNotFoundException extends AuthenticationException {
    public SucceedOAuthUserNotFoundException(String msg, Throwable t) {
        super(msg, t);
    }

    public SucceedOAuthUserNotFoundException(String msg) {
        super(msg);
    }
}

File: src/main/kotlin/com/wafflestudio/draft/security/oauth2/client/exception/SucceedOAuthUserNotFoundException.kt
package com.wafflestudio.draft.security.oauth2.client.exception

import org.springframework.security.core.AuthenticationException
import kotlin.jvm.Throws
import com.wafflestudio.draft.service.FCMService
import com.wafflestudio.draft.service.FCMInitializer
import com.wafflestudio.draft.security.oauth2.client.KakaoOAuth2Client
import com.wafflestudio.draft.security.oauth2.client.FacebookOAuth2Client
import com.wafflestudio.draft.security.JwtTokenProvider

class SucceedOAuthUserNotFoundException : AuthenticationException {
    constructor(msg: String?, t: Throwable?) : super(msg, t) {}
    constructor(msg: String?) : super(msg) {}
}